packages/create-starter/src/index.ts

```diff
@@ -56,6 +57,8 @@ export async function main() {
     process.exit(1);
   }
 
+  await trackSelectedKit(options.kit)
```

Comment:
I don't think it's gonna be a good user experience to force the CLI to wait for the tracking before it's gonna go with the cloning of the starter kit. Maybe we could extract the remainder of the function after this call into a separate function and call them in parallel?

```suggestion
  const [createStarterResult] = await Promise.allSettled([
    createStarter(/* necessary params */), 
    trackSelectedKit(options.kit)
  ])
  if (createStarterResult.status === 'rejected') {
    const err = createStarterResult.reason
    console.log(red(err instanceof Error ? err.message : 'Creating starter kit failed')))
    process.exit(1)
  }
```